docs/sources/next/javascript-api/k6-experimental/fs/file/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'File'
+excerpt: 'File represents a file with methods for reading, seeking, and obtaining file stats.'
+weight: 10
+weight: 10
+---
+
+# File
+
+The `File` class represents a file with methods for reading, seeking, and obtaining file stats. It's returned by the [open](link-to-open-doc) function.
```

Comment:
Is this `link-to-open-doc` a placeholder?

Reply:
I forgot it behind 👍🏻 The new linking system relies on absolute URLs, which is a bit of a pain, so I put placeholders while I write, and make a pass once I'm done to fill the links. Obviously forgot that one, so I might need a better system...